manifests/runtime.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: codefresh.io/v1alpha1
 kind: Runtime
 metadata:
-  name: "{{ name }}"
-  namespace: "{{ namespace }}"
+  name: '{{ name }}'
+  namespace: '{{ namespace }}'
 spec:
-  defVersion: 1.0.1
-  version: 0.0.510
+  defVersion: 2.0.0
```

Comment:
remind me what this is doing? something with upgrade right?

Reply:
yeah. all the binaries that are out there currently have `maxDefVersion = "1.0.1"`, which means that if they will encounter a `runtime.yaml` with a higher version, they will immediately fail the installation, and let the user know they should upgrade their cli-v2 binary.
user can still install using the `--version <whatever_old_version>` if they *really* want to.

but only newer binaries will know how to handle definitions with version 2.0.0 (which now include the `feature` field.